Suite2p/suite2p_parameters.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, danse_params: dict):
         self.paths: dict          = danse_params.get(defs.Parameters.Main.PATHS, {})
         self.params: dict         = danse_params.get(defs.Parameters.Main.PARAMETERS, {})
         self.preview_params: dict = danse_params.get(defs.Parameters.Main.PREVIEW, {})
-        self.time_length: int      = self.get_time_length()
+        self.time_length: int     = self.get_time_length()
+        self.is_microscope: bool  = (self.paths[defs.Parameters.Path.H5PATH][0].split('/')[-1] == defs.DoricFile.Deprecated.Dataset.IMAGES_STACK)
```

Comment:
I would pass it as a parameter from danse

Reply:
Yeah. I can just thought having this information show in the attribute would be useless so I removed it.

Reply:
We don't have to save it in attributes, same as we don't save all the parameters that we pass from the operation `View` to the `Process`